### PR TITLE
Add message that granting permissions takes time.

### PR DESCRIPTION
**Pull request by stephenmichaelf**
Fix for #723

### CODE REVIEW

**Comment by Cristie** [comment]
@stephenmichaelf  Re: fix for #773 "What's going wrong?" Returns a HTTP 401 Not Authorized User

File: src/Agent.Listener/Configuration/WindowsServiceControlManager.cs [hunk 1]
```diff
@@ -142,6 +142,7 @@ private void GrantDirectoryPermissionForAccount(string accountName)
             string workFolder = IOUtil.GetWorkPath(HostContext);
             Directory.CreateDirectory(workFolder);
             Trace.Info(StringUtil.Format("Set full access control to group for the folder {0}", workFolder));
+            Trace.Info(StringUtil.Loc("GrantingFilePermissions"));
```
Comment (TingluoHuang) [inline src/Agent.Listener/Configuration/WindowsServiceControlManager.cs]: I think we would like do `Terminal.write` instead of `Trace` from the caller. :)

**Comment by stephenmichaelf** [comment]
@TingluoHuang Oops :) Fixed. That makes more sense.

File: src/Misc/layoutbin/en-US/strings.json [hunk 2]
```diff
@@ -300,6 +300,7 @@
   "GCUnusedTrackingFile": "Mark tracking file '{0}' for GC, since it hasn't been used for {1} days.",
   "GenerateAndRunUpdateScript": "Generate and execute update script.",
   "GetSources": "Get Sources",
+  "GrantingFilePermissions": "Granting file permissions may take some time.",
```
Comment (TingluoHuang) [inline src/Misc/layoutbin/en-US/strings.json]: what about just `Granting file permissions to '{0}'`, {0} will be the account name?

**Comment by stephenmichaelf** [comment]
@TingluoHuang fixed!

**Pull request merged**